Mathlib/Algebra/Associated.lean

```diff
@@ -60,6 +60,12 @@ theorem dvd_of_dvd_pow (hp : Prime p) {a : α} {n : ℕ} (h : p ∣ a ^ n) : p 
   exact ih dvd_pow
 #align prime.dvd_of_dvd_pow Prime.dvd_of_dvd_pow
 
+theorem dvd_mul {a b : α} : p ∣ a * b ↔ p ∣ a ∨ p ∣ b :=
```

Comment:
Should the name contain `iff`? (Not sure.)

Reply:
Only if we also change `Nat.Prime.dvd_mul`